jormungandr/src/network/subscription.rs

```diff
@@ -306,15 +321,18 @@ impl Sink<net_data::Fragment> for FragmentProcessor {
                     Error::new(Code::Internal, e)
                 });
             } else {
-                ready!(self.poll_send_fragments(cx));
+                ready!(self.poll_send_fragments(cx))?;
             }
         }
     }
 }
 
 impl FragmentProcessor {
     fn poll_send_fragments(&mut self, cx: &mut Context<'_>) -> Poll<Result<(), Error>> {
-        ready!(self.mbox.poll_ready(cx));
+        let logger = &self.logger;
+        ready!(self.mbox.poll_ready(cx))
+            .map_err(|e| debug!(logger, "error sending fragment"; "reason" => %e))
+            .ok();
```

Comment:
Bubble up this one as well. It's good to do logging here in a centralized place. Clarify the message as "error sending fragments for processing" because it's an internal channel.